### ISSUE TITLE
复现结果比原始论文好

### ISSUE THREAD

**Issue by TianSongS** [BUPT-SN/ETM#2]
我自己实现了llama3_features_tensor.py，同时实现了test.py 测试代码。使用tran.py训练了2个epoch，在测试集上效果比原始论文要好，不知道为啥？
测试集
===== Test Set Results =====
Test Accuracy for I-E: 0.8524 | Test F1 for I-E: 0.7084
Test Accuracy for N-S: 0.8519 | Test F1 for N-S: 0.6016
Test Accuracy for T-F: 0.8271 | Test F1 for T-F: 0.8233
Test Accuracy for J-P: 0.7043 | Test F1 for J-P: 0.6984

验证集
Validation Loss: 0.0431
Accuracy for I-E: 0.8501, F1 Score for I-E: 0.6963
Accuracy for N-S: 0.8790, F1 Score for N-S: 0.6569
Accuracy for T-F: 0.8386, F1 Score for T-F: 0.8383
Accuracy for J-P: 0.7037, F1 Score for J-P: 0.6998

**Comment by BUPT-SN** [comment]
Thank you for the reminder! I’ve uploaded get_llama3_feature.py to obtain Llama 3 embeddings. please give it another try.

**Comment by TianSongS** [comment]
Using your newly updated script, I reproduced your work on the Kaggle dataset and was surprised to find that the performance was even better than what was reported in the paper. After training for 6 epochs, the validation accuracy reached 

Validation Loss: 0.0673                                                                                                                                                                                         Accuracy for I-E: 0.8484, F1 Score for I-E: 0.6931                                                                                                                                                              Accuracy for N-S: 0.8945, F1 Score for N-S: 0.6242                                                                                                                                                              Accuracy for T-F: 0.8375, F1 Score for T-F: 0.8517                                                                                                                                                              Accuracy for J-P: 0.7735, F1 Score for J-P: 0.8253  

Since the code for evaluating accuracy on the test set was not provided, I conducted the performance evaluation on the test set separately.
===== Test Set Results =====
Test Accuracy for I-E: 0.8530 | Test F1 for I-E: 0.6760
Test Accuracy for N-S: 0.8888 | Test F1 for N-S: 0.6446
Test Accuracy for T-F: 0.8329 | Test F1 for T-F: 0.8567
Test Accuracy for J-P: 0.8138 | Test F1 for J-P: 0.8417
Avg Test F1: 0.7547538954958875

Apart from the file paths, I did not make any changes to the original code.

**Comment by BUPT-SN** [comment]
Thank you for reproducing the experiment and sharing these detailed results. With the same training code and hyperparameter settings, some variation in results is perfectly normal. This also demonstrates that the model is easy to train and can achieve performance exceeding the other baselines.
